interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -0,0 +1,412 @@
+ *
+ * In this view all therapy groups are listed with their details and links to their details screen.
+ *
+ * Copyright (C) 2016 Shachar Zilbershlag <shaharzi@matrix.co.il>
+ * Copyright (C) 2016 Amiel Elboim <amielel@matrix.co.il>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Shachar Zilbershlag <shaharzi@matrix.co.il>
+ * @author  Amiel Elboim <amielel@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+?>
+<?php require 'header.php'; ?>
+<span class="hidden title"><?php echo xlt('Therapy Group Finder');?></span>
+<div id="therapy_groups_list_container" class="container">
+
+    <!--------- ERRORS ----------->
+    <?php if($deletion_try == 1 && $deletion_response['success'] == 0) :?>
+        <div class="row">
+            <div class="col-md-6 col-md-offset-3">
+                <div class="alert alert-danger text-center">
+                    <p class="failed_message"><?php echo xlt($deletion_response['message']);?></p>
+                </div>
+            </div>
+        </div>
+    <?php endif ?>
+
+    <!---------- FILTERS SECTION ------------->
+    <button id="clear_filters" class="btn"><?php echo xlt("Clear Filters")?></button>
+    </br></br></br>
+    <div id="filters">
+        <div class="row">
+            <div class=" form-group col-md-2">
+                <label class="" for="group_name_filter"><?php echo xlt('Group Name');?>:</label>
+                <input type="text" class="form-control" id="group_name_filter" placeholder="" >
+            </div>
+            <div class=" form-group col-md-2">
+                <label class="" for="group_id_filter"><?php echo xlt('Group Id');?>:</label>
+                <input type="number" class="form-control" id="group_id_filter" placeholder="" >
+            </div>
+            <div class=" form-group col-md-2">
+                <label class="" for="group_type_filter"><?php echo xlt('Group Type');?>:</label>
+                <select type="text" class="form-control" id="group_type_filter" placeholder="" >
+                    <option value=""><?php echo xlt('choose');?></option>
+                    <?php foreach ($group_types as $type):?>
+                        <option value="<?php echo attr($type);?>"><?php echo xlt($type) ;?></option>
```

Comment:
If this list and the lists below are already translated, then don't need to do it here.